public/doc/01_icd.md

```diff
@@ -1303,7 +1303,45 @@ FAQ
   A: No, there are no packets. Since the packet transmission rate is adaptive to the payload bit rate, if the payload is 0, the packet rate is 0 as well.
   You will have to feed data to the local streaming interface to generate packets on the network. There is a debug application that can be used to generate dummy video data.
 
-
+### Systemd Watchdog
```

Comment:
Maybe use the "@section" thing header of other sections of this file? I guess that there's some doxygen trick in place (but we can't test, as documentation build is disabled =/ ).

Reply:
So I guess we want to leave this all the way at the end of the doc ? Adding it elsewhere feels like breaking the continuity of the doc.

Reply:
Maybe before the FAQ? FAQ being the last one makes sense.

Reply:
I'll skip the @section part, b/c then this will become a section of the last _chapter_, which is _Registries_ in this case. So, I'll just make it a new chapter before FAQ.

Reply:
Chapter? Really? Where is this defined? Guess I did something wrong on my PR adding stuff here... hehe. Anyway, yep, it should be on the same level as FAQ.

Reply:
Sorry I meant the L1 header :D, couldn't find the right word at the moment.

Reply:
Done !